src/js/themes/base.d.ts

```diff
@@ -943,6 +943,58 @@ export interface ThemeType {
     colors?: GraphColorsType;
     extend?: ExtendType;
   };
+  number?: {
```

Comment:
I'm wondering if we can shorten the all number type of definition by leveraging TextProps.